platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -308,6 +308,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapView:(MGLMapView *)mapView didChangeUserTrackingMode:(MGLUserTrackingMode)mode animated:(BOOL)animated;
 
+/**
+ Returns a CGPoint value to offset the user location annotation, relative to
+ the camera's center coordinate.
```

Comment:
A screen point can only be relative to another screen point, not a geographic coordinate. I think the intention is to say that it’s relative to the center of the map view. Note that the navigation SDK’s analogous `NavigationMapViewDelegate.navigationMapViewUserAnchorPoint(_:)` method is expressed relative to the map view’s origin, not its center. There are pros and cons to either approach, so it’s good that the frame of reference is being called out here.

Reply:
>A screen point can only be relative to another screen point, not a geographic coordinate.

Does this mean the camera's center can also relate to a CGPoint instead of a coordinate?

Reply:
I think I'm going to base my implementation off of the map view's origin as well, to keep things consistent with the implementation in the Navigation SDK.